index.html

```diff
@@ -484,14 +488,146 @@ <h4><code>readproperty</code></h4>
+        </p>
+        <p>Upon successfully submitting the value of the requested <a>Property</a>, the Thing MUST send a message to the
+          requesting <code>Consumer</code> containing the following members:</p>
+        <table class="def">
+          <caption>Members of a <code>writeproperty</code> response message</caption>
+          <thead>
+            <tr>
+              <th>Member</th>
+              <th>Type</th>
+              <th>Assignment</th>
+              <th>Description</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td><code>messageType</code></td>
+              <td>string</td>
+              <td>"response"</td>
+              <td>A string which denotes that this message is a response sent from a <a>Thing</a> to a <a>Consumer</a>.
+              </td>
+            </tr>
+            <tr>
+              <td><code>operation</code></td>
+              <td>string</td>
+              <td>"writeproperty"</td>
+              <td>A string which denotes that this message relates to a <code>writeproperty</code> operation.
+              </td>
+            </tr>
+            <tr>
+              <td><code>name</code></td>
+              <td>string</td>
+              <td>Mandatory</td>
+              <td>The name of the <a>Property</a> being written, as per its key in the <code>properties</code> member of
+                the
+                <a>Thing Description</a>.
+              </td>
+            </tr>
+            <tr>
+              <td><code>value</code></td>
+              <td>any</td>
+              <td>Optional</td>
+              <td>The value which has been successfully assigned to the <a>Property</a> being written, with a format
+                conforming to the data schema of the corresponding PropertyAffordance in the <a>Thing Description</a>.
+              </td>
+            </tr>
+            <tr>
+              <td><code>timestamp</code></td>
+              <td>string</td>
+              <td>Optional</td>
+              <td>A timestamp in date-time format [[RFC3339]] set to the time the property write took place.</td>
+            </tr>
+          </tbody>
+        </table>
+        <pre class="example" title="writeproperty response message">
+          {
+            "thingID": "https://mythingserver.com/things/mylamp1",
+            "messageID": "db25fe4f-bee8-43a7-8ff0-3a1ff6e620b0",
+            "messageType": "response",
+            "operation": "writeproperty",
+            "name": "on",
+            "value": true,
+            "timestamp": "2024-01-13T23:20:50.52Z",
+            "correlationID": "f6cf46a8-9c96-437e-8b53-925b7679a990"
+          }
+        </pre>
+        <p>If the <a>Thing</a> can confirm that the requested value has been set successfully then the response message
+          to a <code>writeproperty</code> request SHOULD contain a <code>value</code> member with its value set to the
```

Comment:
I think some informative explanation/example about setting a less precise value can be used. The Consumer tries to write `3.1415` but the Thing doesn't have that precision so it replies with `3.14`

Reply:
I think this is OK in principle, but doesn't it get a bit complicated if the data schema uses the `multipleOf` member in a data schema to define a particular level of precision? That might result in an error case instead.

I've added an assertion about this, but may expand upon it when defining error conditions later.